pkg/vendir/fetch/http/sync.go

```diff
@@ -48,6 +48,7 @@ func (t *Sync) Sync(dstPath string, tempArea ctlfetch.TempArea) (ctlconf.LockDir
 
 	incomingTmpPath := filepath.Dir(tmpFile.Name())
 	archivePath := filepath.Join(incomingTmpPath, path.Base(t.opts.URL))
+	tmpFile.Close()
```

Comment:
Since we are actively closing the file here, can we remove the defer from line 41 and copy this line to the if statement in line 45?

This way, we will not try to close the same file twice.

Reply:
true, I made the change